public/src/game objects/graphics/fill rect.js

```diff
@@ -1,26 +1,31 @@
-var config = {
-    width: 800,
-    height: 600,
-    type: Phaser.WEBGL,
-    parent: 'phaser-example',
-    scene: {
-        create: create
+class FillRect extends Phaser.Scene
+{
+    constructor ()
+    {
+        super();
     }
-};
-
-var game = new Phaser.Game(config);
+    create ()
```

Comment:
Careful with the spacing between sections (i.e. add a blank line between `preload` and `create` methods.